### PR TITLE
feat(integrations): make sure stacktrace-link feature is set

### PR DESCRIPTION
**Pull request by scefali**
Before:
Screenshot 2023-11-20 at 3 26 00 PM

After:
Screenshot 2023-11-20 at 3 26 13 PM

### CODE REVIEW

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -1597,6 +1597,8 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
     "organizations:integrations-opsgenie": True,
     # Enable one-click migration from Opsgenie plugin
     "organizations:integrations-opsgenie-migration": False,
+    # Enable stacktrace linking
+    "organizations:integrations-stacktrace-link": True,
```
Comment (ceorourke) [inline src/sentry/conf/server.py:1601]: is this a precursor to removing the flag?

**Review by ceorourke — APPROVED** [review]

File: src/sentry/conf/server.py (hunk 1 above)
Comment (scefali) [reply src/sentry/conf/server.py:1601]: @ceorourke No, the flag stays. We don't read it except in the integration features UI to show it's available.

Comment (ceorourke) [reply src/sentry/conf/server.py:1601]: :( feels like there should be a better way to remove flags for features that are ... not feature gated for anyone

Comment (scefali) [reply src/sentry/conf/server.py:1601]: @ceorourke yea probably someone on the future Ecosystem team can think about how to handle these cases better

**Pull request merged**